pkg/cloudprovider/providers/vsphere/vclib/diskmanagers/vdm.go

```diff
@@ -0,0 +1,72 @@
+package diskmanagers
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/golang/glog"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
+	"k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere/vclib"
+)
+
```

Comment:
Added one line description for each.

`// virtualDiskManager implements VirtualDiskProvider Interface for creating and deleting volume using VirtualDiskManager`


`// vmDiskManager implements VirtualDiskProvider interface for creating volume using Virtual Machine Reconfigure approach`